### DISCUSSION TITLE
Pre-sale question

### DISCUSSION THREAD

**Discussion by Swalabrr** [revk/xESP32-Faikout#137] [General]
Hi!

I consider buying 2 Faikins in order to control my 2 Daikin ACs. They both have the BRP069C4x WiFi module installed, but the AC units are slightly different versions. One is FTXM71R2V1B and one is FTXM35R2V1B. I have a few questions:

1. Obviously, if Faikin is compatible with them. In your docs, it seems that they are, but would like a confirmation. Also, for my units, is it more complicated than populating S21 with Faikin device?

2. These indoor units have a PIR motion detector embedded. Is it possible to read their status using Faikin? In their app or in HomeAssistant (HACS plugin), these motion sensors are not available, unfortunately.

3. Is it possible to lock the flaps position to "left/right" or "up/down" intentionally using Faikin? At times, I'd rather have the unit blowing air into a specific direction, but all I can do right now is to "block" them manually in the desired direction.

4. I understand that for Home Assistant integration, it has to connect through MQTT. I don't know much about MQTT, but i already have a MQTT in HA and its not clear to me how I can connect a secondary device using MQTT. Can you please, point me in the right direction?

Thank you,
Dan

**Comment by revk** [comment]
I can't say if it will work with specific models - others may be able to confirm.

It can connect to the MQTT in HA.

My own experience of the vents is you can turn on and off moving and stop them where you want. I am not aware of any command or status in S21 to set to specific point.

The use of PIR may be on/off as well, possibly. I think it may be the "comfort" setting.

Bear in mind that Daikin don't publish this so it has all been reverse engineered and trial and error. Also not everything is necessarily even possible via the S21. But we do have various dumps and debug via MQTT that can (and has) allow some new features to be discovered and added over time.

**Comment by Swalabrr** [comment]
thanks for the quick reply!

guess i'll bit the bullet and see where this goes :D. I'm tired of them being randomly disconnected because of cloud integration.

**Reply by revk** [reply]
Good luck - if you have S21 it will almost certainly "work", and maybe if there are new features they can be worked out in future.
